src/lib/watch-processor.ts

```diff
@@ -32,6 +32,14 @@ export function queueRecordKey(obj: KubernetesObject) {
   return strat === "singular" ? `${kind}/${ns}` : `${kind}/${name}/${ns}`;
```

Comment:
```suggestion
  return strat === "singular" ? `${kind}/${ns}` : `${kind}/${name}/${ns}`;
```

This is technically a new strategy, we can protect users by having a strategy for just `kind`, allowing them to choose.

Reply:
Yep, currently thinking "kind", "kindNs", "kindNsName", and "global" as a good, default set.  Wrote-up the proposal for those in the follow-on Issue, here: https://github.com/defenseunicorns/pepr/issues/1126 .

Hope that covers all our bases. :crossed_fingers: